src/fedora/39/amd64/Dockerfile

```
@@ -66,4 +66,5 @@ RUN dnf --setopt=install_weak_deps=False install -y \
 # Dependencies for Aspnetcore
 RUN cd ~ && \ 
     curl -sL https://rpm.nodesource.com/setup_20.x
-RUN dnf install -y nodejs && dnf clean all
\ No newline at end of file
+RUN dnf install -y nodejs && dnf clean all
+ENV NO_UPDATE_NOTIFIER
```

I think this needs to be set to a value.
```suggestion
ENV NO_UPDATE_NOTIFIER=true
```

---

According to the docs it doesn't need to be set to any value but it doesn't harm: https://www.npmjs.com/package/update-notifier#user-settings